flatdata-generator/flatdata/generator/templates/rust/structure.jinja2

```diff
@@ -56,11 +83,6 @@ impl<'a> {{ struct.name }}Ref<'a> {
 
     {% endif %}
     {% endfor %}
-
```

Comment:
Unrelated to change?

Reply:
Actually it is. It removed dead code in Rust.